packages/opencensus-core/src/trace/types.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as modelTypes from './model/types';
+import * as samplerTypes from './sampler/types';
+import * as configTypes from './config/types';
+import * as exportersTypes from '../exporters/types';
+
+/** Main interface for tracing. */
+export interface Tracing {
```

Comment:
> Is Tracing implemented anywhere?

Yes, it is implemented in nodejs package. You can check the current implementation in our dev branch.

> I think this follows the same rule as Tracer: there doesn't need to be an interface because it doesn't make sense to have multiple implementations.

I think my previous comment on the need for a Tracer interface is valid here. I would like to keep both interfaces.